### PR TITLE
feature : add modal

### PR DESCRIPTION
**Pull request by maskpa100**
Сделал модальное окно покаместь.

### CODE REVIEW

File: ui/ui-kit/Modal/Modal.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+type ModalType = {
+    active: boolean
+    setActive: any
+    title: string
+    children: any
+  }
+
+const Modal: React.FC<ModalType> = ({active, setActive, title, children}) => {
+    const classs = `${s.modal} ${s.effect}`
```
Comment (TURP1) [inline ui/ui-kit/Modal/Modal.tsx:11]: try using the class name library, without it it's harder to understand the style logic

File: pages/dev/ui-kit/Modal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  }
+}
+
+const Modals = () => {
+    const [ModalActive, setModalActive] = useState(false);
+  return (
+    <PageWrapper>
+        <button className={style.openModal} onClick={() => {setModalActive(true)}}>Modal open</button>
+        <Modal active={ModalActive} setActive={setModalActive} title="Email sent">
+          <div className={style.text}>We have sent a link to confirm your email to epam@epam.com</div>
```
Comment (TURP1) [inline pages/dev/ui-kit/Modal/index.tsx:19]: I would hide this logic inside component

Comment (TURP1) [reply pages/dev/ui-kit/Modal/index.tsx:19]: maybe it`s better to use prop text

File: pages/dev/ui-kit/Modal/index.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+          <div className={style.buttonBloc}>
+            <button className={style.button}>
```
Comment (TURP1) [inline pages/dev/ui-kit/Modal/index.tsx:21]: what is this button for?

**Review by irinrudko — APPROVED** [review]
agree with @TURP1 's comments

**Review by ilich01 — APPROVED** [review]

**Review by Viarcinski — APPROVED** [review]
looks good

**Comment by TURP1** [comment]
resolve conflicts

File: pages/dev/ui-kit/index.tsx [hunk 4]
```diff
@@ -42,7 +45,20 @@ const Login = () => {
 
             </div>
         </div>
-
+        <div className={style.kitBlock}>
+            <h2 className={style.componentHeader}>Modal</h2>
+            <div className={style.modal}>
+            <button className={style.open} onClick={() => {setModalActive(true)}}>Modal open</button>
```
Comment (TURP1) [inline pages/dev/ui-kit/index.tsx:51]: we already have Button component, you can use it

File: @ui/ui-kit/Modal/Modal.tsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+import s from "./modal.module.scss"
+
+type ModalType = {
+    active: boolean
+    setActive: any
+    title: string
+    children: any
+  }
+
+const Modal: React.FC<ModalType> = ({active, setActive, title, children}) => {
```
Comment (TURP1) [inline @ui/ui-kit/Modal/Modal.tsx:11]: what about translation text for modal?

File: @ui/ui-kit/Modal/Modal.tsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    const open = active ? classNames(s.modal, s.effect, s.show) : classNames(s.modal, s.effect)  
+    return (
+    <>
+        <div className={open} >
+            <div className={s.content}>
+                <div className={s.item}>
+                    <div className={s.title}>{title}t</div>
```
Comment (TURP1) [inline @ui/ui-kit/Modal/Modal.tsx:18]: <div className={s.title}>{title}**t**</div>
you have lost a letter 't'

File: @ui/ui-kit/Modal/Modal.tsx (hunk 5 above)
Comment (TURP1) [inline @ui/ui-kit/Modal/Modal.tsx:11]: what about title translation prop?

**Pull request closed without merging**